releasenotes/notes/bump-jmxfetch-to-0_49_1-618bbc38451cf0d8.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Upgraded JMXFetch to `0.49.1 <https://github.com/DataDog/jmxfetch/releases/tag/0.49.1>`_ which adds support for
+    ZGC Major and Minor Cycles and ZGC Major and Minor Pauses beans.
```

```suggestion
    ZGC Major and Minor Cycles, and ZGC Major and Minor Pauses beans.
```
Comma for better readability.